global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview-010088",
+    "dotnet": "3.0.100-preview4-010391",
```

Comment:
What is enforcing that the relevant tests will actually run against the version of CoreCLR included in this version of the SDK?

If you don't have this version installed locally, various tests may have a different behavior when running or building things using the `dotnet.exe` that is on your path vs the one that is restored by `build.cmd`

Reply:
> If you don't have this version installed locally

I do have it installed locally.